src/status_im2/contexts/syncing/view.cljs

```diff
@@ -4,51 +4,70 @@
             [quo2.foundations.colors :as colors]
             [react-native.core :as rn]
             [status-im2.contexts.syncing.sheets.sync-device-notice.view :as sync-device-notice]
-            [status-im2.contexts.syncing.styles :as styles]
+            [status-im2.contexts.syncing.style :as style]
             [utils.re-frame :as rf]))
+;;TODO remove mock data (#https://github.com/status-im/status-mobile/issues/15142)
+(defn navigation-bar
+  []
+  [rn/view {:style style/navigation-bar}
+   [quo/page-nav
+    {:align-mid?   true
+     :mid-section  {:type :text-only :main-text ""}
+     :left-section {:type                :blur-bg
+                    :icon                :i/arrow-left
+                    :icon-override-theme :dark
+                    :on-press            #(rf/dispatch [:navigate-back])}}]])
 
 (defn render-device
-  [device-name device-status]
-  [:<>
-   [rn/view {:style styles/device-row}
-    [quo/icon-avatar
-     {:size  :medium
-      :icon  :i/placeholder
-      :color :primary
-      :style {:margin-vertical 10}}]
-    [rn/view {:style styles/device-column}
-     [quo/text
-      {:accessibility-label :device-name
-       :weight              :medium
-       :size                :paragraph-1
-       :style               {:color colors/neutral-100}} device-name]
+  [{:keys [device
+           this-device?
+           type]}]
+  [rn/view {:style style/device-container}
+   [rn/view {:style style/icon-container}
+    [quo/icon
+     (if (= type :mobile) :i/mobile :i/desktop)
 
-     [quo/text
-      {:accessibility-label :device-status
-       :weight              :regular
-       :size                :paragraph-2
-       :style               {:color colors/neutral-50}} device-status]]]])
+     {:color colors/white}]]
+   [rn/view {:style style/device-details}
+    [quo/text
+     {:accessibility-label :device-name
+      :weight              :medium
+      :size                :paragraph-1
+      :style               {:color colors/white}} device]
+    [quo/text
+     {:accessibility-label :next-back-up
+      :size                :paragraph-2
+      :style               {:color colors/white-opa-40}} "Next backup in 04:36:12"]
```

Comment:
It may be a good idea to add a red border or something around this dummy text, since there's no actual backup and the text is not translated.

After the closing map there's a missing a line break :)

Reply:
done! 
and I added the red border of doom/shame/incompleteness :)